### PR TITLE
IA 148 Gateway For Member

### PR DESCRIPTION
**Pull request by AlexWeinstein92**

### CODE REVIEW

File: src/main/riddl/Gateway/MemberGateway/memberStories.riddl [hunk 1]
```diff
@@ -24,11 +24,11 @@ story RegisterMember is {
...
         briefly "App tells the Gateway to enroll the member",
-    //   step from actor GatewayAPI "Register Member" to entity ImprovingApp.MemberContext.Member
+    //   step from actor GatewayAPI "Register Member" to entity ImprovingApp.MembersContext.Member
     //     briefly "add new organization",
-    //   step from entity ImprovingApp.MemberContext.Member "MemberRegistered" to pipe ImprovingApp.MemberContext.MemberEvents
+    //   step from entity ImprovingApp.MembersContext.Member "MemberRegistered" to pipe ImprovingApp.MembersContext.MemberEvents
     //     briefly "organization added"
-    //   step from pipe ImprovingApp.MemberContext.MemberEvents "MemberEnrolled" to actor App
+    //   step from pipe ImprovingApp.MembersContext.MemberEvents "MemberEnrolled" to actor App
     }
```
Comment (reid-spencer) [inline src/main/riddl/Gateway/MemberGateway/memberStories.riddl:23]: Wouldn't it be better also to uncomment these so riddlc can validate the pathIds and the documentation can represent the relationships they imply?

File: src/main/riddl/Gateway/actorDefinitions.riddl [hunk 2]
```diff
@@ -1,3 +1,3 @@
 actor Member is "The person that is being enrolled"
 actor App is "The applicaiton the user is using to perform this action"
```
Comment (reid-spencer) [inline src/main/riddl/Gateway/actorDefinitions.riddl:2]: I would prefer this was specified as an Application definition. I think this was written before the Application/Epic/Actor triangle was established.

File: src/main/riddl/Gateway/actorDefinitions.riddl [hunk 3]
```diff
@@ -1,3 +1,3 @@
...
-actor GatewayAPI is "The interface that accepts the request for processing by the backend"
\ No newline at end of file
+actor GatewayAction is "The interface that accepts the request for processing by the backend"
```
Comment (reid-spencer) [inline src/main/riddl/Gateway/actorDefinitions.riddl:3]: Similar this needs to be written as a Gateway Context

File: src/main/riddl/Gateway/gateway.riddl [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
-    include "MemberGateway/memberStories.riddl"
-    include "NotificationGateway/notificationStories.riddl"
-    include "OrderGateway/orderStories.riddl"
-    include "PlaceGateway/placeStories.riddl"
-    include "ProductGateway/productStories.riddl"
-    include "ReservationGateway/reservationStories.riddl"
-    include "StoreGateway/storeStories.riddl"
-    include "TenantGateway/tenantStories.riddl"
-    include "VendorGateway/vendorStories.riddl"
+    //include "actorDefinitions"
```
Comment (reid-spencer) [inline src/main/riddl/Gateway/gateway.riddl]: Why comment this out?

File: src/main/riddl/Gateway/gatewayStories.riddl [hunk 5]
```diff
@@ -15,11 +15,11 @@ story RegisterMember is {
...
       step from actor App send command Member.EnrollMember to actor GatewayAPI
         briefly "send creation message",
-    //   step from actor GatewayAPI "Register Member" to entity ImprovingApp.MemberContext.Member
+    //   step from actor GatewayAPI "Register Member" to entity ImprovingApp.MembersContext.Member
     //     briefly "add new organization",
-    //   step from entity ImprovingApp.MemberContext.Member "MemberRegistered" to pipe ImprovingApp.MemberContext.MemberEvents
+    //   step from entity ImprovingApp.MembersContext.Member "MemberRegistered" to pipe ImprovingApp.MembersContext.MemberEvents
     //     briefly "organization added"
-    //   step from pipe ImprovingApp.MemberContext.MemberEvents "MemberEnrolled" to actor App
+    //   step from pipe ImprovingApp.MembersContext.MemberEvents "MemberEnrolled" to actor App
```
Comment (reid-spencer) [inline src/main/riddl/Gateway/gatewayStories.riddl]: Can we uncomment those and get the compiler and hugo documentation to see them?

**Review by reid-spencer — CHANGES_REQUESTED** [review]
I've left some questions on various constructs for you to consider.

**Review by reid-spencer — DISMISSED** [review]
LGTM

**Review by JamesYopp — APPROVED** [review]
Looks good

**Pull request merged**